packages/devtools_app/test/memory_screen_test.dart

```diff
@@ -604,68 +619,53 @@ void main() {
 
     // OneClass hilighted.
     await downArrow(autoCompletes4.indexOf('OneClass'));
-    await pumpAndSettleOneSecond();
+    await pumpAndSettleTwoSeconds();
 
     // Show's auto-complete dropdown with the 2nd item highlighted.
-    await expectLater(
-      find.byType(MemoryBody),
-      matchesGoldenFile(
-          'goldens/allocation_dropdown_hilight_line_2_golden.png'),
+    await checkGolden(
+      'goldens/allocation_dropdown_hilight_line_2_golden.png',
+      key: searchAutoCompleteKey,
     );
-    // Await delay for golden comparison.
-    await pumpAndSettleTwoSeconds();
 
     // OneMoreClass hilighted.
     await downArrow(autoCompletes4.indexOf('OneMoreClass'));
     await pumpAndSettleOneSecond();
 
     // Show's auto-complete dropdown with the 3rd item highlighted.
-    await expectLater(
-      find.byType(MemoryBody),
-      matchesGoldenFile(
-          'goldens/allocation_dropdown_hilight_line_3_golden.png'),
+    await checkGolden(
+      'goldens/allocation_dropdown_hilight_line_3_golden.png',
+      key: searchAutoCompleteKey,
     );
-    // Await delay for golden comparison.
-    await pumpAndSettleTwoSeconds();
 
     // SecondClass hilighted.
     await downArrow(autoCompletes4.indexOf('SecondClass'));
     await pumpAndSettleOneSecond();
```

Comment:
you are now waiting 2+1 =3 seconds for some of these check golden cases. I would remove the pumpAndSettleOneSecond calls or remove the two second delay from check goldens and just add the delay you need where you need it.

Reply:
They weren't needed I think this was part of early testing. They're gone.